Tuturial-Structure-Template.md

```diff
@@ -1,68 +1,178 @@
-# Title #
-[header image]
+>The title describes a task that you want to do. For example: **Implement step-up authentication**.
+> 
+> To write a page, use the style and tone suggest in the style guide.
 
-## Intro ##
-> (do not include section title) 
+# Title
 
-Write a paragraph that introduces the topic. Explain why the content is relevant, and make sure to introduce the concepts you plan to cover.
- 
-## Prerequisites ##
-> (use as section title)
+> Explain why the content is relevant and what the user will achieve.
+> 
+> For example:
 
-List what the reader needs to know, as well as packages or SDKS that must be available, to successfully work through the tutorial. Keep the list short, but be sure to include language and concepts the reader should be familiar with.  
+Step-up authentication provides a higher level of assurance that a person is who they claim to be. When a user tries to perform a task, they are challenged to produce additional forms of authentication. In this case, when a user who is already logged into your Agora client tries to connect to a highly secure conference, the back-end system sends a one-time sms-code to a registered mobile device which the user enters in your Agora Client.
 
-**IMPORTANT**: The last prerequisite will always be a link to the _“How to get started with Agora”_ tutorial using the url:
-```
-https://www.agora.io/en/blog/how-to-get-started-with-agora?utm_source=medium&utm_medium=blog&utm_campaign=
-```
+This page shows you how to add this functionality to you Agora Client and security infrastructure. 
 
-Append your tutorial’s title to the end of the link above, with an underscore replacing any spaces in the title, as in this example:
+## About the tech
```

Comment:
Is the name for each chapter fixed, or we can use different names? As About the tech Vs. Understand the tech

Reply:
Be best if the title is fixed. I've changed to Understand because I prefer it as well. Thanks